sequencer/api/migrations/postgres/V502__epoch_drb_and_root.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE epoch_drb_and_root (
+    epoch BIGINT PRIMARY KEY,
+    drb_result BYTEA,
+    block_header BYTEA,
```

Comment:
think migration will fail with this extra comma at the end
```suggestion
    block_header BYTEA
```